src/main/java/com/creativemd/littletiles/client/render/LittleTilesBlockRenderHelper.java

```diff
@@ -37,51 +36,26 @@ public static void renderCubes(IBlockAccess world, ArrayList<CubeObject> cubes,
         IBlockAccessFake fake = (IBlockAccessFake) extraRenderer.blockAccess;
         fake.world = renderer.blockAccess;
 
-        for (int i = 0; i < cubes.size(); i++) {
-
-            if (cubes.get(i).icon != null) renderer.setOverrideBlockTexture(cubes.get(i).icon);
-
-            if (cubes.get(i).block != null) if (cubes.get(i).meta != -1) {
+        for (CubeObject cube : cubes) {
```

Comment:
Probably better to stick with the c style iterator here, just cache the size().  This is an ArrayList so I think it'll allocate an iterator (vs an Array which won't).

My assumption is this is in the hot path (rendering) so we'll want to minimize allocations.

Reply:
do a c style loop and final Cube cube = cubes.get(i);

Reply:
fixed